scalafix-core/src/main/scala/scalafix/internal/patch/PatchInternals.scala

```diff
@@ -191,6 +200,20 @@ object PatchInternals {
     loop(patch)
   }
 
+  // don't decompose Atomic Patch
+  def foreachPatchUnit(patch: Patch)(f: Patch => Unit): Unit = {
```

Comment:
this is a very generic signature, yet it's used only for accumulation, maybe the 2 new methods could be merged?

Reply:
I use it twice